opensafely/check.py

```diff
@@ -103,10 +105,16 @@ def get_repository_name():
     else:
         git_config_path = Path(".git", "config")
         if not git_config_path.is_file():
+            print("Git config file not found")
             return
         config = configparser.ConfigParser()
-        config.read(git_config_path)
+        try:
+            config.read(git_config_path)
+        except Exception as e:
+            print(f"Unable to read git config.\n{str(e)}")
```

Comment:
I'm not sure we should print this level of detail for the drive by check, so maybe only print these if continue_on_error is False?